controllers/auth.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const { validate } = require('indicative').validator;
```

Comment:
We should be using freedom-indicative already.

Reply:
Used the `freedom-indicative`.